src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-tutorial.mdx

```diff
@@ -0,0 +1,476 @@
+/>
+
+Ready to get started? Let's go!
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="first-query"
+    title="1. Your first query"
+  >
+  <SideBySide>
+  <Side>
+Let's start by looking at a single event type in NRDB called **Transaction**, gathered by New Relic APM. Every NRQL query must have `SELECT` and `FROM` clauses: you must `SELECT` some data and tell us where it's `FROM`. Start with this basic query that selects everything from the **Transaction** event type.
+ 
+    ```sql
+    SELECT *
+    FROM Transaction
+    ```
+
+  </Side>
+  <Side>
+    <img
+  title="Select and From query"
+  alt="A screenshot of a query with Select and From"
+  src={queriesnrqltutorial1}
+/>
+  </Side>
+  </SideBySide>
+  <SideBySide>
+  <Side>
+ This returns a lot of results, each with a timestamp and a collection of attributes. For now, we only want a single result, so let's limit the results to a single record using `LIMIT 1`.
+
+ <Callout variant="important"> When a LIMIT isn't supplied the default will be used, which is 100 table rows for SELECT * queries or 10 aggregated values for FACET queries and SELECT (attributes) queries. You can specify any limit up to the maximum. Use LIMIT MAX to return the maximum number of results possible.
+</Callout>
+
+    ```sql
+    SELECT *
+    FROM Transaction
+    LIMIT 1
+    ```
+  </Side>
+  <Side>
+ <img
+  title="Select, From, and Limit query"
+  alt="A screenshot of a query with Select, From, and Limit"
+  src={queriesnrqltutorial2}
+/>
+  </Side>
+</SideBySide>
+<SideBySide>
+  <Side>
+    We're now controlling the volume of the results. But what if we don't want all the attributes? What if we would prefer to see only specific data points? Like SQL, we can do this with only a few characters by replacing `*` with the name of the attribute(s) we want. In this case, we'll ask for the name of a transaction and the duration of time it took.
+
+    ```sql
+    SELECT name, duration
+    FROM Transaction
+    ```
+    
+  </Side>
+  <Side>
+  <img
+  title="Select and From with Duration query"
+  alt="A screenshot of a query with Select and From using Duration"
+  src={queriesnrqltutorial3}
+/>
+  </Side>
+</SideBySide>
+With this, you've completed queries that can return either all or some specific attributes of an event, as well as querying a specific number of events. Let's go to the next step: aggregating data.
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="aggregate-data"
+    title="2. Aggregate data"
+  >
+<SideBySide>
+  <Side>
+Sometimes you may want to view summarized aggregate data across many events instead of viewing individual events. This is where NRDB shines. NRDB can scan billions of events and provide instant, real-time answers to questions about your data. For example, each `Transaction` event has a duration attribute that represents how many seconds the call took to execute. Let's see if we can discover the average duration across all `Transaction` events:
```

Comment:
instead of 'event', I think we should use 'data points'. The reason being: some of the data types we can query with NRQL include `Metric` and `Span` and `Log` data, and we avoid calling those 'events' (trying to keep those MELT data types separate). So even though we often internally refer to everything in NRDB as 'events', we should try as much as we can to avoid comingling those terms. So instead I think it's just as easy to say 'NRDB can scan billions of data points...' and such.

Reply:
Sounds good to me! Thank you!